backend/routes/assetlinks.ts

```diff
@@ -0,0 +1,27 @@
+import { Request, Response, Router } from 'express';
+
+const assetLinksRouter = Router();
+
+/**
+ * tl:dr: Serve the static file needed for mobile deep linking
```

Comment:
`tl:dr:` implies that this will be preceded or proceeded by a more verbose explanation.  Since there is none, how do you feel about dropping it (and leaving "Serve the ...")?